.ci/Jenkinsfile

```
@@ -30,6 +30,7 @@ pipeline {
   }
   triggers {
     issueCommentTrigger('(?i).*(?:jenkins\\W+)?run\\W+(?:the\\W+)?tests(?:\\W+please)?.*')
+    cron 'H 02 * * *'
```

better to put a range of hours, the CI will decide the less overload time.
```suggestion
    cron 'H H(2-5) * * *'
```